src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Date.txt

```diff
@@ -1,58 +1,27 @@
-﻿>> Text(Date(-1, 2, 3))
-/*SqlRunner*/ Compile Error
+﻿>> Date(-1, 2, 3)
 #Error
 
->> Date(-1, 2, 3)
-#Error
 
 >> Date(Blank(), 2, 3)
 Blank()
 
 // dates before 1900 get converted to post-1900
->> Text(Date(80, 1, 2))
-/*SqlRunner*/ Compile Error
-"1/2/1980"
+>> Date(80, 1, 2)
+Date(1980,1,2)
 
 // months and days overflow (but not for SQL)
->> Text(Date(2000, 25, 69))
-/*SqlRunner*/ Compile Error
-"3/10/2002"
-
->> Text(Date(2000, 2, -1))
-/*SqlRunner*/ Compile Error
-"1/30/2000"
-
->> Text(Date(2000, -2, 1))
-/*SqlRunner*/ Compile Error
-"10/1/1999"
-
->> Text(Date(2000, -2, -1))
-/*SqlRunner*/ Compile Error
-"9/29/1999"
+>> Date(2000, 25, 69)
+Date(2002,3,10)
```

Comment:
This is nice, and should help running tests in other locales as well

Reply:
Also important because Sql doesn't support Text(date), so needed to update tests.